client/extensions/woocommerce/state/sites/settings/stripe-connect-account/reducer.js

```diff
@@ -43,7 +51,49 @@ function connectAccountCreateComplete( state = {}, action ) {
 	} );
 }
 
+/**
+ * Updates state to indicate account (details) fetch request is in progress
+ *
+ * @param  {Object} state  Current state
+ * @return {Object}        Updated state
+ */
+function connectAccountFetch( state = {} ) {
+	return Object.assign( {}, state, {
```

Comment:
This looks like the same object from connectAccountCreate.

Reply:
It is. I've removed this create reducer from this PR and will DRY it in #19256